example.v0.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{refsheet}
+
+% Taken from https://tex.stackexchange.com/a/99784.
+
+% Title area
+\usepackage{titling} % Allows for use of date, author, etc. after \maketitle
+% Ref: http://tex.stackexchange.com/questions/3988/titlesec-versus-titling-mangling-thetitle
+\let\oldtitle\title
+\renewcommand{\title}[1]{\oldtitle{#1}\newcommand{\mythetitle}{#1}}
+\renewcommand{\maketitle}{%
+{\begin{center}\Large \mythetitle\end{center}}
+}
+
+% Code
+\usepackage{listings}
+\lstset{language=[LaTeX]TeX,%
+  basicstyle=\itshape,%
+  keywordstyle=\normalfont\ttfamily,%
+  morekeywords={part,chapter,subsection,subsubsection,paragraph,subparagraph}%
+}
+
+\usepackage{lipsum}
+
+\title{\LaTeXe Cheat Sheet}
```

Comment:
Does this not need `\LaTeXe\ Cheat Sheet`?

Reply:
Maybe, or actually probably `\LaTeXe{} Cheat Sheet`, but again I did not change much of the example I pulled in. It was the original implementation.